cmd/domains.go

```diff
@@ -25,8 +26,14 @@ func DomainsList(appID string, results int) error {
 
 	fmt.Printf("=== %s Domains%s", appID, limitCount(len(domains), count))
 
+	var domainNames []string
```

Comment:
Same comment here as [above](https://github.com/deis/workflow-cli/pull/144/files#r72125711). Introducing a `Domains` object into the SDK with the applicable sorting functions required would be the optimal approach, IMO.